File: android-wrapper/app/src/main/java/org/m0skit0/android/hms/unity/ads/RewardAdStatusListener.java
package org.m0skit0.android.hms.unity.ads;

import com.huawei.hms.ads.reward.Reward;

public interface RewardAdStatusListener {
    void onRewardAdClosed();
    void onRewardAdFailedToShow(final int errorCode);
    void onRewardAdOpened();
    void onRewarded(final Reward reward);
}

File: android-wrapper/app/src/main/java/org/m0skit0/android/hms/unity/ads/RewardAdStatusListener.kt
package org.m0skit0.android.hms.unity.ads

import com.huawei.hms.ads.reward.Reward

interface RewardAdStatusListener {
    fun onRewardAdClosed()
    fun onRewardAdFailedToShow(errorCode: Int)
    fun onRewardAdOpened()
    fun onRewarded(reward: Reward)
}